### PR TITLE
TF Review fix documentation

### PR DESCRIPTION
**Pull request by marinsalinas**
Hello @tracypholmes, here are the changes for items 2, 3, 4, 5, on #25.

Thanks!

### CODE REVIEW

File: website/docs/d/database_users.html.markdown [hunk 1]
```diff
@@ -63,9 +63,9 @@ In addition to all arguments above, the following attributes are exported:
...
 Block mapping a user's role to a database / collection. A role allows the user to perform particular actions on the specified database. A role on the admin database can include privileges that apply to the other databases as well.
 
-**NOTE:** The available privilege actions for custom MongoDB roles support a subset of MongoDB commands. See Unsupported Commands in M10+ Clusters for more information.
+-> **NOTE:** The available privilege actions for custom MongoDB roles support a subset of MongoDB commands. See Unsupported Commands in M10+ Clusters for more information.
 
-**IMPORTANT** If a user is assigned a custom MongoDB role, they cannot be assigned any other roles.
+~> **IMPORTANT:** If a user is assigned a custom MongoDB role, they cannot be assigned any other roles.
 
 * `name` - Name of the role to grant.
 * database_name -  Database on which the user has the specified role. A role on the `admin` database can include privileges that apply to the other databases.
```
Comment (themantissa) [inline website/docs/d/database_users.html.markdown]: Missing backticks (but the other one looks fixed!)

File: website/docs/r/cloud_provider_snapshot.html.markdown [hunk 2]
```diff
@@ -35,7 +35,8 @@ resource "mongodbatlas_cloud_provider_snapshot" "test" {
 
 In addition to all arguments above, the following attributes are exported:
 
-* `id` - Unique identifier of the snapshot.
+* `id` - Terraform Unique ID for this resource, no confundir con el snapshot_id.
```
Comment (themantissa) [inline website/docs/r/cloud_provider_snapshot.html.markdown]: I think we got some Spanish mixed in here.  Do not confuse with snapshot_id ? :)

File: website/docs/r/project.html.markdown [hunk 3]
```diff
@@ -3,7 +3,7 @@ layout: "mongodbatlas"
 page_title: "MongoDB Atlas: project"
 sidebar_current: "docs-mongodbatlas-resource-project"
 description: |-
```
Comment (themantissa) [inline website/docs/r/project.html.markdown]: Something weird still in the formatting - when looking at the rich diff the header still looks odd.

File: website/docs/d/database_users.html.markdown (hunk 1 above)
Comment (PacoDw) [reply website/docs/d/database_users.html.markdown]: Thank you, I will attend it 👍

File: website/docs/r/cloud_provider_snapshot.html.markdown (hunk 2 above)
Comment (PacoDw) [reply website/docs/r/cloud_provider_snapshot.html.markdown]: This issue was attended with on the #28 pr

**Pull request merged**